api/api.go

```diff
@@ -511,6 +513,8 @@ type CloudBackupStatusRequest struct {
 	// ignored. This could be GroupCloudBackupId too, and in that case multiple
 	// statuses belonging to the groupCloudBackupID is returned.
 	ID string
+	// Indicates if this is for nearsync
+	NearSyncBackupRestore bool
```

Comment:
Should we rename this to just `NearSync`? The CloudBackupStatusRequest is used for both backups and restores, and this field does not need to clarify it even further